searchhotel.html

```diff
@@ -0,0 +1,303 @@
+        <fieldset class="list-filter-group">
+          <legend class="list-filter-title">Инфраструктура:</legend>
+          <ul class="list-filter-items">
+            <li class="list-filter-item">
+              <label class="control">
+                <input class="visually-hidden control-input" type="checkbox" name="pool" checked>
+                <span class="control-mark"></span>
+                <span class="control-label">Бассейн</span>
+              </label>
+            </li>
+
+            <li class="list-filter-item">
+          <label class="control">
+            <input class="visually-hidden control-input" type="checkbox" name="parking" checked>
+            <span class="control-mark"></span>
+            <span class="control-label">Парковка</span>
+          </label>
+            </li>
+
+            <li class="list-filter-item">
+          <label class="control">
+            <input class="visually-hidden control-input" type="checkbox" name="wi-fi">
+            <span class="control-mark"></span>
+            <span class="control-label">Wi-Fi</span>
+          </label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="list-filter-group">
+          <legend class="list-filter-title">Тип жилья:</legend>
+          <ul class="list-filter-items">
+            <li class="list-filter-item">
+              <label class="control">
+                <input class="visually-hidden control-input" type="radio" name="houseroom-group" value="hotel" checked>
+                <span class="control-mark"></span>
+                <span class="control-label">Гостиница</span>
+              </label>
+            </li>
+
+            <li class="list-filter-item">
+          <label class="control">
+            <input class="visually-hidden control-input" type="radio" name="houseroom-group" value="motel">
+            <span class="control-mark"></span>
+            <span class="control-label">Мотель</span>
+          </label>
+            </li>
+
+            <li class="list-filter-item">
+          <label class="control">
+            <input class="visually-hidden control-input" type="radio" name="houseroom-group" value="apartments">
+            <span class="control-mark"></span>
+            <span class="control-label">Апартаменты</span>
+          </label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="list-filter-group">
+          <legend class="list-filter-title">Стоимость, ₽:</legend>
+            <button class="range-toggle toggle-min" type="button">
+              <span class="visually-hidden">Изменить минимальную цену</span>
+            </button>
+            <button class="range-toggle toggle-max" type="button">
+              <span class="visually-hidden">Изменить максимальную цену</span>
+            </button>
+            <label for="list-filter-label">
+              <input class="list-filter-input" type="number" name="min-price">от
```

Comment:
```suggestion
              <input class="list-filter-input" type="number" name="min-price" min="0" max="10000" value="0">
              от
```

Для второго свой `value`